src/libstd/keyword_docs.rs

```diff
@@ -710,3 +789,79 @@ mod loop_keyword { }
 /// [book]: ../book/ch05-01-defining-structs.html
 /// [reference]: ../reference/items/structs.html
 mod struct_keyword { }
+
+#[doc(keyword = "super")]
+//
+/// The parent of the current [module].
+///
+/// [module]: ../../nightly/book/ch07-02-defining-modules-to-control-scope-and-privacy.html
+mod super_keyword { }
+
+#[doc(keyword = "trait")]
+//
+/// A single interface shared across types.
+mod trait_keyword { }
+
+#[doc(keyword = "true")]
+//
+/// A value of type `bool` representing logical **true**.
+mod true_keyword { }
+
+#[doc(keyword = "type")]
+//
+/// Give a new name to an existing type.
+mod type_keyword { }
+
+#[doc(keyword = "unsafe")]
+//
+/// Opt out of statically enforced memory safety.
+mod unsafe_keyword { }
+
+#[doc(keyword = "use")]
+//
+/// Import or rename items from other crates or modules.
+mod use_keyword { }
+
+#[doc(keyword = "where")]
+//
+/// Add bounds to a type parameter.
+mod where_keyword { }
+
+#[doc(keyword = "while")]
+//
+/// Loop while a condition remains true.
```

Comment:
```suggestion
/// Loop while a condition is upheld.
```
In lieu of https://github.com/rust-lang/rust/issues/53667.